### DISCUSSION TITLE
senior front end dev

### DISCUSSION THREAD

**Discussion by ghost** [WeAreMahsaAmini/Core#122] [Introductions]
Hey everyone, I'm a senior front end dev (react js) and I can help with anything front end related.

**Comment by sinahnyazdi** [comment]
Can you help with this? #124

**Reply by ghost** [reply]
Sure, no problem, any existing repo, or should I just jump right into it now?  :D

**Reply by sinahnyazdi** [reply]
I didn't create it yet, could you create it and let me know 🚀 🚀

**Reply by ghost** [reply]
on it...🔥

**Comment by Upward-Spiral** [comment]
Senior FE dev as well. Would be happy to help. (React,Vue,Angular,Typescript)

**Reply by sinahnyazdi** [reply]
Thanks alot!, @realavawalker will share the repo with us, so we can work on it together.
